calibration/CTDMOG/CGINS-CTDMOG-13601__20150612.csv

```diff
@@ -22,4 +22,4 @@ serial,name,value,notes
 37-13601,CC_ptempa0,169.1845,
 37-13601,CC_ptempa1,-0.06413934,
 37-13601,CC_ptempa2,-1.14163e-06,
-37-13601,CC_wbotc,7.7547e-07,
+37-13601,CC_wbotc,7.754676e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT